lib/bugsnag/loggers/multi_logger.rb

```diff
@@ -0,0 +1,85 @@
+module Bugsnag::Loggers
```

Comment:
I dont think we should implement this. We can recommend another one if we want, but its out of the scope of our lib imo